app/models/manageiq/providers/openstack/cinder_manager_mixin.rb

```diff
@@ -0,0 +1,20 @@
+module ManageIQ::Providers::Openstack::CinderManagerMixin
+  extend ActiveSupport::Concern
+  include ::CinderManagerMixin
+
+  included do
+    # TODO: how about many storage managers???
+    # Should use has_many :storage_managers,
+    has_one  :cinder_manager,
+             :foreign_key => :parent_ems_id,
+             :class_name  => "ManageIQ::Providers::StorageManager::CinderManager",
+             :autosave    => true,
+             :dependent   => :destroy
```

Comment:
hm, I think we were removing the :dependent => :destroy lately https://github.com/ManageIQ/manageiq-providers-openstack/pull/208/files

We might want to drop it?

Reply:
Hm, though this passes the CI, so could you maybe do that as separate PR? We might need to backport both then.